build/l10n.js

```diff
@@ -70,6 +70,8 @@
     if (!containsFetchableLocale) {
       document.documentElement.dataset.noCompleteBug = true;
     }
+
+    L10n.registerLocales.call(this);
```

Comment:
I would have to access meta.defaultLanguage|availableLanguages from build/l10n.js. Wanted to avoid having to expose that.